app/package/controllers.py

```diff
@@ -330,70 +331,66 @@ def purge_data_package(publisher, package):
         return handle_error('GENERIC_ERROR', e.message, 500)
 
 
-@package_blueprint.route("/<publisher>/<package>/finalize",
-                         methods=["POST"])
+@package_blueprint.route("/upload", methods=["POST"])
 @requires_auth
-@is_allowed('Package::Create')
-def finalize_metadata(publisher, package):
+def finalize_metadata():
     """
     DPR metadata finalize operation.
```

Comment:
Don't use the acronym DPR in the codebase if you can.

The first line here is also a pointless piece of documentation as it tells the reader nothing they don't already know from the method name. I would just delete this and keep the second line "This API is ..." which **is** useful.